hw/top_earlgrey/chip_earlgrey_verilator.cc

```diff
@@ -10,13 +10,13 @@
 #include "verilator_sim_ctrl.h"
 
 int main(int argc, char **argv) {
```

Comment:
Move this also to `hw/top_earlgrey/dv/verilator`?

Reply:
Yes, it is common to have the name of this file aligned with the SV wrapper and store it next to it. So I'd move it to hw/top_earlgrey/dv/tb/chip_earlgrey_verilator_tb.cc (or whatever name we end up with).